engines/bastion_katello/app/assets/javascripts/bastion_katello/content-views/details/filters/available-errata-filter.controller.js

```diff
@@ -83,6 +83,12 @@ angular.module('Bastion.content-views').controller('AvailableErrataFilterControl
             nutupane.refresh();
         };
 
+        $scope.$watch('rule.date_type', function (value) {
```

Comment:
> Better to define a function that is called via ng-change on the radio input and handles this logic than register a watcher.

I know you like ng-change better. But this watch is only needed if "errata-filters by ids is needed."